x-pack/legacy/plugins/uptime/public/pages/monitor.tsx

```diff
@@ -56,13 +56,7 @@ export const MonitorPage = ({ setBreadcrumbs }: MonitorPageProps) => {
         dateEnd={absoluteDateRangeEnd}
```

Comment:
We are still passing `absoluteDateRangeStart/End` to `MonitorStatusDetails` in this page. We can probably delete the code getting those values, and reference the query hook inside the component instead.

Reply:
Agreed, done